stravalib/client.py

```diff
@@ -29,6 +29,24 @@
 logging.captureWarnings(True)
 
 
+def warn_param_deprecation(param_name: str):
```

Comment:
@jsamoocha this is great to have functions that handle our warnings. Do you think they belong in the exc.py module rather than the client module as helpers?

Reply:
Ah, yes - good suggestion! The `exc` module makes more sense. I moved them there.